release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1669,6 +1669,147 @@ If these values are not defined in a failure domain configuration, for instance
+
+* Previously, when installing a cluster on {ibm-power-server-title} the required security group rules were not created. With this release, the missing security group rules for installation are identified and created and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-43518[*OCPBUGS-43518*])
+
+* Previously, when you tried to add a worker node with the `oc adm node-image` command by using an instance that was previously created with {rh-openstack-first}, the operation failed. With this release, the issue is resolved by correctly setting the user-managed networking configuration.(link:https://issues.redhat.com/browse/OCPBUGS-43513[*OCPBUGS-43513*])
+
+* Previously, when destroying a cluster on {gcp-first} a forwarding rule incorrectly blocked the installer. As a consequence, the destroy process failed to complete. With this release, the issue is resolved by the installer setting its state correctly and marking all destroyed resources as deleted. (link:https://issues.redhat.com/browse/OCPBUGS-42789[*OCPBUGS-42789*])
+
+* Previously, when configuring the Agent-Based Installer installation in a disconnected environment with more than one mirror for the same source, the installation might fail. This occurred because one of the mirrors was not checked. With this release, all mirrors are used when multiple mirrors are defined for the same source and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-42705[*OCPBUGS-42705*])
+
+* Previously, you could not change the `AdditionalTrustBundlePolicy` parameter in the `install-config.yaml` file for the Agent-based Installer. The parameter was always set to `ProxyOnly`. With this release, you can set `AdditionalTrustBundlePolicy` to other values, for example, `Always`. By default, the parameter is set to `ProxyOnly`. (link:https://issues.redhat.com/browse/OCPBUGS-42670[*OCPBUGS-42670*])
+
+* Previously, when you installed a cluster and tried to add a worker node with the `oc adm node-image` command it failed because the date, time, or both might have been inaccurate. With this release, the issue is resolved by applying the same Network Time Protocol (NTP) configuration in the target cluster `MachineConfig` chrony resource to the node ephemeral live environment (link:https://issues.redhat.com/browse/OCPBUGS-42544[*OCPBUGS-42544*])
```

Comment:
```suggestion
* Previously, when you installed a cluster and tried to add a compute node with the `oc adm node-image` command, it failed because the date, time, or both might have been inaccurate. With this release, the issue is resolved by applying the same Network Time Protocol (NTP) configuration in the target cluster `MachineConfig` chrony resource to the node ephemeral live environment (link:https://issues.redhat.com/browse/OCPBUGS-42544[*OCPBUGS-42544*])
```